modules/combined/test/tests/frictional_contact/single_point_2d/tests

```diff
@@ -5,6 +5,7 @@
     exodiff = 'single_point_2d_out.e'
     custom_cmp = 'single_point_2d.cmp'
     superlu = true
+    compiler = 'GCC CLANG'
```

Comment:
Why only gcc and clang?  We should say why.

Reply:
We do not have PETSc3.7.6 on intel. There is no way to test it.

Reply:
Wouldn't it be better to just have petsc_version = '>=3.7.6' in the tests file then?  That way, it would get skipped if you were on a platform with an older petsc (intel).

Reply:
I'd prefer to see us start using the "NOT" syntax moving forward. Try
compiler = `!INTEL`